templates/config/nginx/common/locations.conf

```diff
@@ -0,0 +1,76 @@
+# NGINX CONFIGURATION FOR COMMON LOCATION
+# DO NOT MODIFY, ALL CHANGES LOST AFTER UPDATE EasyEngine (ee)
+# Basic locations files
+location = /favicon.ico {
+  access_log off;
+  log_not_found off;
+  expires max;
+}
+
+location = /robots.txt {
+  # Some WordPress plugin gererate robots.txt file
+  # Refer #340 issue
+
+  return 200 'User-agent: *
+Disallow: /';
+  access_log off;
+  log_not_found off;
+}
+# Cache static files
+location ~* \.(ogg|ogv|svg|svgz|eot|otf|woff|mp4|ttf|rss|atom|jpg|jpeg|gif|png|ico|zip|tgz|gz|rar|bz2|doc|xls|exe|ppt|tar|mid|midi|wav|bmp|rtf|swf)$ {
+  add_header "Access-Control-Allow-Origin" "*";
+  access_log off;
+  log_not_found off;
+  expires max;
```

Comment:
@mbtamuli this `location` block is not recommended on dev sites. Static file caching should be only used in production.

Reply:
We'll configure this based in the next release which will have [--dev|--prod] flags or something similar, to specify if it is a site in development or production.